### ISSUE TITLE
`GoogleObjectProperty.prototype.readGoogle` fails for `this._cfg.name === 'optimized'`

### ISSUE THREAD

**Issue by anulman** [huafu/ember-google-map#78]
Specifically, it seems to be due to `googleObject` not having a function `getIsOptimized`. Confirmed in reference [here](https://developers.google.com/maps/documentation/javascript/3.exp/reference#Marker).

I've tried setting `marker.isOptimized = true` when declaring markers, but the problem persisted. I've worked around this for now (on deadline!) by commenting out the relevant line and will be pushing a fork shortly; please let me know if a PR will be helpful.

For convenience, here are the relevant fn && my kludge of a fix:

```javascript
// ember-google-map/addon/core/google-object-property.js
  /**
   * Reads the value from the given google object
   *
   * @method readGoogle
   * @param {google.maps.MVCObject} googleObject
   * @returns {Object}
   */
  GoogleObjectProperty.prototype.readGoogle = function (googleObject) {
    debugger;
    var val;
    if (this._cfg.read) {
      val = this._cfg.read.call(this, googleObject);
    } else if (this._cfg.optionOnly) {
      return Object.create(null);
    } else {
      val = googleObject['get' + capitalize(this._cfg.name)]();
    }
    return this.fromGoogleValue(val);
  };
```

```javascript
// ember-google-map/app/views/google-map/marker.js
  googleProperties: {
    isClickable: {name: 'clickable', event: 'clickable_changed'},
    isVisible:   {name: 'visible', event: 'visible_changed'},
    isDraggable: {name: 'draggable', event: 'draggable_changed'},
    // isOptimized: {name: 'optimized', readOnly: true},
    title:       {event: 'title_changed'},
    opacity:     {cast: helpers.cast.number},
    icon:        {event: 'icon_changed'},
    zIndex:      {event: 'zindex_changed', cast: helpers.cast.integer},
    map:         {readOnly: true},
    'lat,lng':   {
      name:       'position',
      event:      'position_changed',
      toGoogle:   helpers._latLngToGoogle,
      fromGoogle: helpers._latLngFromGoogle
    }
  },
```

**Comment by mjc** [comment]
This affects my branch as well.

**Comment by huafu** [comment]
fixed in `0.0.23`
